vhdbuilder/packer/list-images.sh

```diff
@@ -3,9 +3,31 @@ set -euxo pipefail
 
 CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-docker}"
 
+function generate_image_bom_for_containerd() {
+    temp_image_bom=/opt/azure/containers/temp-image-bom.json
+    IFS_backup=$IFS
+    IFS=$'\n'
+    ctr_list=$(ctr --namespace k8s.io image list | sed 1d | awk '{print $1, $3}')
+    crictl_list=$(crictl images --no-trunc | sed 1d | awk '{printf "%s:%s %s\n", $1,$2,$3;}')
+
+    for image in $ctr_list; do
+        tag=$(echo $image | awk '{print $1}')
+        # skip image tags that consist of a sha256 hash
+        [[ $tag == sha256* ]] && continue
+        digest=$(echo $image | awk '{print $2}')
+        id=$(echo "$crictl_list" | grep -e "$tag" | awk '{print $2}')
+
+        jq --arg repoTag "$tag" --arg repoDigest "$digest" --arg id "$id" -n '{id:$id, repoTags:[$repoTag], repoDigests:[$repoDigest]}' >> $temp_image_bom
+    done
+
+    IFS=$IFS_backup
+    jq --slurpfile images $temp_image_bom -n '$images | group_by(.id) | map({id:.[0].id, repoTags:map(.repoTags | add) | unique, repoDigests:map(.repoDigests | add) | unique})' > /opt/azure/containers/image-bom.json
+    rm -f $temp_image_bom
+    jq < /opt/azure/containers/image-bom.json
```

Comment:
why input?

Reply:
oh it's written above, this is just validation I suppose?

Reply:
yeah was mainly for validation/so we can see the output during the build, through I'm seeing now this line might be producing errors during test builds...

Reply:
ended up removing altogether